website/docs/source/v2/provisioning/ansible_common.html.md

```diff
@@ -51,6 +51,30 @@ Some of these options are for advanced usage only and should not be used unless
 
   By default, this option is disabled and Vagrant generates an inventory based on the `Vagrantfile` information.
 
+- `galaxy_command` (template string) - The command pattern used to install Galaxy roles when `galaxy_role_file` is set.
+
+  The following placeholders can be used in this command pattern:
```

Comment:
I think you want to indent everything here two more spaces so it falls under the bullet instead of breaking the list